### PR TITLE
Removed set of env var to anable/disable HTTP on the bridge

### PR DESCRIPTION
**Pull request by ppatierno**
### Type of change

- Refactoring

### Description

The coming bridge (0.23.0) doesn't have AMQP support anymore.
For this reason, we removed the possibility to enable/disable HTTP because it doesn't make sense anymore and it has to be always active. See https://github.com/strimzi/strimzi-kafka-bridge/pull/694 for more details.
This PR removes the handling of the `KAFKA_BRIDGE_HTTP_ENABLED` on the `KafkaBridge` CR that was used for this purpose.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [x] Make sure all tests pass
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally

### CODE REVIEW

**Review by im-konge — APPROVED** [review]

**Comment by ppatierno** [comment]
We will need the new bridge release to test it, before merging.

**Review by scholzj — APPROVED** [review]
LGTM. But if we cannot merge it until we have the new Bridge version, it should be Draft or maybe closed and reopened together with the bridge upgrade?

**Comment by ppatierno** [comment]
Thanks! Converted to draft.

**Comment by im-konge** [comment]
@ppatierno `0.23.1` Bridge is released and it's part of operators repository, should this be changed to open now?

**Comment by ppatierno** [comment]
@im-konge yes, I have just rebased and will run regression.

**Pull request merged**
